http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/HttpRoutesSpec.scala

```diff
@@ -443,124 +438,6 @@ class HttpRoutesSpec
     }
   }
 
-  "RuntimeRoutesV2" should "create azure runtime" in {
-    Post(s"/api/v2/runtimes/${workspaceId.value.toString}/azure/azureruntime1")
-      .withEntity(ContentTypes.`application/json`,
-                  defaultCreateAzureRuntimeReq.asJson.spaces2
-      ) ~> routes.route ~> check {
-      status shouldEqual StatusCodes.Accepted
-      validateRawCookie(header("Set-Cookie"))
-    }
-  }
-
-  it should "reject azure runtime with invalid workspaceId" in {
-    Post(s"/api/v2/runtimes/invalidWorkspaceId/azure/azureruntime1")
-      .withEntity(ContentTypes.`application/json`,
-                  defaultCreateAzureRuntimeReq.asJson.spaces2
-      ) ~> routes.route ~> check {
-      status shouldEqual StatusCodes.BadRequest
-      responseEntity.toStrict(5 seconds).futureValue.data.utf8String should include(
-        "Invalid workspace id invalidWorkspaceId, workspace id must be a valid UUID"
-      )
-    }
-  }
-
-  it should "reject azure runtime with invalid runtimeName" in {
-    Post(s"/api/v2/runtimes/${workspaceId.value.toString}/azure/invalidRuntime")
-      .withEntity(ContentTypes.`application/json`,
-                  defaultCreateAzureRuntimeReq.asJson.spaces2
-      ) ~> routes.route ~> check {
-      status shouldEqual StatusCodes.BadRequest
-      responseEntity.toStrict(5 seconds).futureValue.data.utf8String should include(
-        "Invalid runtime name invalidRuntime"
-      )
-    }
-  }
-
-  it should "get an azure runtime" in {
-    Get(s"/api/v2/runtimes/${workspaceId.value.toString}/azure/azureruntime1") ~> routes.route ~> check {
-      status shouldEqual StatusCodes.OK
-      responseAs[GetRuntimeResponse].clusterName shouldBe RuntimeName("azureruntime1")
-      validateRawCookie(header("Set-Cookie"))
-    }
-  }
-
-  it should "404 on get azure runtime when it does not exist" in {
-    Get(s"/api/v2/runtimes/${workspaceId.value.toString}/azure/fakeruntime") ~> httpRoutes.route ~> check {
-      status shouldEqual StatusCodes.NotFound
-    }
-  }
-
-  it should "delete an azure runtime" in {
-    Delete(s"/api/v2/runtimes/${workspaceId.value.toString}/azure/azureruntime1") ~> routes.route ~> check {
-      status shouldEqual StatusCodes.Accepted
-    }
-  }
-
-  it should "404 on delete azure runtime when it does not exist" in {
-    Delete(s"/api/v2/runtimes/${workspaceId.value.toString}/azure/azureruntime1") ~> httpRoutes.route ~> check {
-      status shouldEqual StatusCodes.NotFound
-    }
-  }
-
-  it should "list runtimes v2 with a workspace" in {
```

Comment:
if we're keeping around listRuntimeV2, should we also keep the tests?

Reply:
Yes, I'll double-check this - there were two listRuntime methods, one with workspace and one without, and we kept only the no-workspace one.